<?xml version="1.0"?>
<launch>

  <include file="$(find very_simple_robot_simulator)/launch/minimal_simulator.launch" />
  <include file="$(find very_simple_robot_simulator)/launch/lidar_simulator.launch" />

  <node pkg="very_simple_robot_simulator" name ="world_state_gui" type ="world_state_gui.py">
   <param name="/map_file" type="yaml" value="$(find vsrs_mapping_pkg)/maps/map.yaml"/>
  </node>
  <node pkg="vsrs_mapping_pkg" name="odom_pix" type="odom_pix.py" output = "screen"/>
  <node pkg="vsrs_mapping_pkg" name="display_map" type="display_map.py" output = "screen"/>
  
  <node pkg="rqt_image_view" name="image_view_debug" type="rqt_image_view"  args="/img_map"/>

</launch>
